solr/core/src/java/org/apache/solr/handler/component/QueryComponent.java

```diff
@@ -421,22 +416,22 @@ public void process(ResponseBuilder rb) throws IOException {
       cmd.setSegmentTerminateEarly(false);
       try {
         if (params.getBool(GroupParams.GROUP_DISTRIBUTED_FIRST, false)) {
-          doProcessGroupedDistributedSearchFirstPhase(rb, cmd, result);
+          doProcessGroupedDistributedSearchFirstPhase(rb, cmd);
           return;
         } else if (params.getBool(GroupParams.GROUP_DISTRIBUTED_SECOND, false)) {
-          doProcessGroupedDistributedSearchSecondPhase(rb, cmd, result);
+          doProcessGroupedDistributedSearchSecondPhase(rb, cmd);
           return;
         }
 
-        doProcessGroupedSearch(rb, cmd, result);
+        doProcessGroupedSearch(rb, cmd);
```

Comment:
subjective/style: if/since we're changing this code area, how about adding a `else { }` wrap and then removing the two `return;` in the first and second phases?
```suggestion
        else {
          doProcessGroupedSearch(rb, cmd);
        }
```